app/models/container_node.rb

```diff
@@ -85,4 +86,22 @@ def evaluate_alert(_alert_id, _event)
     # and these should automaticaly be translated to alerts.
     true
   end
+
+  supports :external_logging_support do
+    unless ext_management_system.respond_to?(:external_logging_route_name)
+      unsupported_reason_add(:external_logging_support, _('This provider type does not support Extrenal Logging'))
+    end
+  end
+
+  def external_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def external_logging_path
+    fqdn = (ipaddress || name || "")
+    hostname = fqdn.split('.').first
```

Comment:
@simon3z The name of the method was misleading so I changed it from `ipaddress` to `kubernetes_hostname`. If it is indeed returning an IP address instead of an hostname then we will add a search for the first octect (OR the full IP). It will not be optimal but won't crush anything and considering that none of our options gave a valid hostname it seems like an acceptable option.

I fixed `hostname` to have a default value of ""